tests/ui/match_single_binding.stderr

```diff
@@ -0,0 +1,14 @@
+error: this match could be written as a `let` statement
+  --> $DIR/match_single_binding.rs:9:5
+   |
+LL | /     match (a, b, c) {
+LL | |         (x, y, z) => {
+LL | |             println!("{} {} {}", x, y, z);
+LL | |         },
+LL | |     }
+   | |_____^ help: try this: `let (x, y, z) = (a, b, c);`
```

Comment:
I'm not happy with the suggestion. It kind of informs you to re-write the `match`-expression as 
```rust
let (x, y, z) = (a, b, c);
println!("{} {} {}", x, y, z);
```
which makes me wonder, why moving the tuple into another one at all. Ideally I'd like to see a help message, that suggest altering the `match` into its body, i.e. the help message should output:
```
help: try this: `println!("{} {} {}", a, b, c);`
```

Reply:
> It kind of informs you to re-write the match-expression as

I agree, the suggestion should be restricted only to the match, not the body.
In fact in this case I wanted to outline only the 2 first lines (but currently unable to), and excluding the body. I will try to find a better solution.

> why moving the tuple into another one at all. Ideally I'd like to see a help message, that suggest altering the match into its body

This example is a simple one and because the body can be very long, i didn't want to deal with it (could be ugly and seems complicated). About the moving, a reason could be the use of another names that bind same variables (or refs or mut ref) to them, such as `let x = &y`.